src/OOVPADatabase/D3D8/4034.inl

```diff
@@ -157,21 +157,29 @@ OOVPA_NO_XREF(D3DDevice_SetTransform, 4034, 13)
 // ******************************************************************
 // * D3D::UpdateProjectionViewportTransform
 // ******************************************************************
-OOVPA_XREF(D3D_UpdateProjectionViewportTransform, 4034, 1 + 9,
+// Generic OOVPA as of 4034 and newer
+OOVPA_XREF(D3D_UpdateProjectionViewportTransform, 4034, 1 + 11,
 
            XREF_D3D_UpdateProjectionViewportTransform,
            XRefOne)
 {
-    // mov esi, XREF_D3DDEVICE
+    // mov e??, XREF_D3DDEVICE
     XREF_ENTRY(0x0B, XREF_D3DDEVICE), // Derived
 
-    // push ebp
-    // mov  ebp, esp
-    OV_MATCH(0x00, 0x55, 0x8B, 0xEC),
-
     // and  esp, 0FFFFFFF0h
     // sub  esp, 58h
     OV_MATCH(0x03, 0x83, 0xE4, 0xF0, 0x83, 0xEC, 0x5C),
+
+    // Start of assembly code group since anything afterward does change over time.
+    // mov e??,
+    OV_MATCH(0x09, 0x8B),
+    // mov eax, [e?? + 0x?]
+    OV_MATCH(0x0F, 0x8B),
+    // fild [ e?? + 0x?]
+    OV_MATCH(0x15, 0xDB),
```

Comment:
Add new line below this since there's 2 instructions between flid and jge.

Reply:
done